src/Serilog.Extensions.Logging.Extensions/Serilog.Extensions.Logging.Extensions.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
New project containing only single file?

Reply:
@Cheesebaron If you want to get rid of `Microsoft.Extensions.Logging/Microsoft.Extensions.DependencyInjection` dependencies then look into MS sources :
```csharp
 internal static class ProviderAliasUtilities
    {
        private const string AliasAttibuteTypeFullName = "Microsoft.Extensions.Logging.ProviderAliasAttribute";

        internal static string GetAlias(Type providerType)
        {
            foreach (CustomAttributeData attributeData in CustomAttributeData.GetCustomAttributes(providerType))
            {
                if (attributeData.AttributeType.FullName == AliasAttibuteTypeFullName)
                {
                    foreach (CustomAttributeTypedArgument arg in attributeData.ConstructorArguments)
                    {
                        Debug.Assert(arg.ArgumentType == typeof(string));

                        return arg.Value?.ToString();
                    }
                }
            }

            return null;
        }
    }
```
😉 I will not tell you exactly why they did that, but since they did it, then the reason was and why not use such a  opportunity?

Reply:
![изображение](https://user-images.githubusercontent.com/21261007/139653836-c91a552d-2666-43ba-a0da-0706f71c6970.png)